txpipe/twopoint.py

```diff
@@ -493,6 +503,18 @@ def call_treecorr(self, i, j, k):
         elif k == POS_POS:
             xx = self.calculate_pos_pos(i, j)
             xtype = sacc.standard_types.galaxy_density_xi
+        elif k == POS_EXT:
```

Comment:
Does this not cause issues if someone tries to run just TXTwopoint on an external, but not the class? i.e. Maybe we ought to move the entire function `call_treecorr` to the extension class

Reply:
What do you think, should we add it to the subclass, or do you want to keep it here? might be worth to get @joezuntz  input once he is back?

Reply:
ok sounds good. I moved it